Makefile

```diff
@@ -13,12 +13,22 @@ LD_FLAGS="-s -w -X $(PACKAGE)/version.Version=$(GIT_VERSION) -X $(PACKAGE)/versi
 
 .PHONY: compile
 compile:
-	go build -o flyte -ldflags=$(LD_FLAGS) ./cmd/
+	@if [ ! -d "cmd/single/dist" ]; then\
+		docker create --name flyteconsole ghcr.io/flyteorg/flyteconsole-release;\
+        docker cp flyteconsole:/app/dist cmd/single;\
+        docker rm -f flyteconsole;\
+    fi
+	go build -tags console -v -o flyte -ldflags=$(LD_FLAGS) ./cmd/
```

Comment:
Can we just quick cleanup the spaces / tabs issues here? Looks like the Makefile uses tabs and here there are some spaces used.

Reply:
updated it, thnaks